derive_builder_core/src/setter.rs

```diff
@@ -256,14 +256,36 @@ mod tests {
         setter.attrs = attrs.as_slice();
         setter.generic_into = true;
         setter.deprecation_notes = &deprecated;
+        setter.try_setter = true;
+
+        // Had to hoist these out to avoid a recursion limit in the quote!
+        // macro invocation below.
```

Comment:
no, problem. You can just add `#![cfg_attr(test, recursion_limit="100")]` to the crate root. That should increase the recursion limit in test-mode only. Just put in a number that's roughly twice as high as the current minimum requirement. :-)